Makefile

```diff
@@ -106,3 +212,66 @@ old-rpm: $(OLD_RPM_SPECFILE) $(RPM_TARBALL)
 	rpmbuild -bb \
 		--define "_topdir $(CURDIR)/rpmbuild" \
 		$(OLD_RPM_SPECFILE)
+
+#
+# Releasing
+#
+
+NEXT_VERSION := $(shell expr "$(VERSION)" + 1)
+
+.PHONY: release
+release:
+	@echo
+	@echo "Checklist for release of osbuild-composer-$(NEXT_VERSION):"
```

Comment:
I don't think this belongs in the Makefile, RELEASE.md would be the place I'd normally look for instructions like this.

Reply:
Can you elaborate? Why does this not belong here? What **does** belong into this makefile? Why is putting this into RELEASE.md preferable? Especially given that this will prevent us from auto-generating the content just as `make release` does.

This target allows running `make release`, returning you generated output that you can easily copy-paste. This is very convenient when prepping a release, as you do not have to edit all the different tags in the commands.

Reply:
Actions belong in the Makefile, instructions belong in textfiles. I'd almost never think to look in the Makefile for the release checklist.